python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -87,7 +83,7 @@ class GangaList(GangaObject):
     _hidden = 1
     _enable_plugin = 1
     _name = 'GangaList'
-    _schema = Schema(Version(1, 0), {'_list': ComponentItem(defvalue=[], doc='The raw list', hidden=1, category='internal'),
+    _schema = Schema(Version(1, 0), {'_list': SimpleItem(defvalue=[], doc='The raw list', hidden=1, category='internal'),
```

Comment:
I'm assuming that nested XML now still work here (I think we have some tests for this but I'm just curious as this _was_ needed a whie ago, but I didn't understand why)